cmd/antrea-agent/agent.go

```diff
@@ -368,8 +368,10 @@ func run(o *Options) error {
 			proxier,
 			k8sClient,
 			nodeRouteController,
-			networkConfig.TrafficEncapMode,
 			nodeConfig,
+			v4Enabled,
+			v6Enabled,
+			networkConfig.TrafficEncapMode.IsNetworkPolicyOnly(),
```

Comment:
I'm ok with adding back v4Enabled / v6Enabled, but I think we should keep passing `networkConfig.TrafficEncapMode` instead of `networkConfig.TrafficEncapMode.IsNetworkPolicyOnly()`

Reply:
May I ask why?

Reply:
I think it keeps the call site simple as `networkConfig.TrafficEncapMode` is readily available and well defined. I don't believe there is value in specializing the parameter too much. That being said, it's a pretty minor point, so feel free to stick to `networkConfig.TrafficEncapMode.IsNetworkPolicyOnly()` if it makes more sense to you or if it makes test cases simpler.

Reply:
thanks, updated